jpa/WFLY-19554_cdi_persistence_ee11.adoc

```diff
@@ -0,0 +1,176 @@
+---
+categories:
+  - cdi-persistence
+stability-level: community
+issue: https://issues.redhat.com/browse/WFLY-19554
+feature-team:
+  developer: Scott Marlow
+  sme:
+   - https://github.com/starksm64
+   - https://github.com/gavinking
+  outside-perspective:
+   -
+promotes:
+promoted-by:
+---
+
+== Overview
+
+Implement the Jakarta Persistence & Jakarta Contexts and Dependency Injection (CDI) integration as per [https://jakarta.ee/specifications/webprofile/11/jakarta-webprofile-spec-11.0-m5#a441]
+
+This section specifies the requirements for Jakarta EE environments that integrate both Persistence and CDI.
+
+Several of the items described in this section are also available with the following (traditional) access methods, as specified in the sections Persistence Unit References and Persistence Context References of [https://jakarta.ee/specifications/platform/11/].
+  - the @Resource annotation with the JNDI lookup attribute.
+  - the JNDI lookup using JNDI API.
+
+The items available with these additional access methods are indicated inline.
+
+3.12.2. Injecting an Entity Manager Factory using CDI [https://jakarta.ee/specifications/webprofile/11/jakarta-webprofile-spec-11.0-m5#injecting-an-entity-manager-factory-using-cdi]
```

Comment:
You didn't copy in section 3.12.1.

Reply:
fixed ^ up with a few other sections more clearly identified.